File: starter_kits/Java/hlt/PlayerId.java
package hlt;

public class PlayerId {
    public final int id;

    public PlayerId(int id) {
        this.id = id;
    }

    @Override
    public String toString() {
        return String.valueOf(id);
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;

        PlayerId playerId = (PlayerId) o;

        return id == playerId.id;
    }

    @Override
    public int hashCode() {
        return id;
    }
}

File: starter_kits/PHP/hlt/PlayerId.php
<?php

namespace hlt;

class PlayerId {
    /**
     * @var int
     */
    public $id;

    public function __construct(int $id)
    {
        $this->id = $id;
    }

    public function __toString()
    {
        return '' . $this->id;
    }

    public function equals($other): bool
    {
        if (!($other instanceof self)) {
            return false;
        }

        return $this->id === $other->id;
    }
}
